docs/kafka/access-mgmt-kafka/README.adoc

```diff
@@ -475,6 +477,37 @@ h|Operations
 |===
 --
 
+[id="proc-user-account-access_{context}"]
+== Granting users access to view other user accounts
+
+[role="_abstract"]
+As an organization administrator, you can use Role-Based Access Control (RBAC) to allow users to view other users in a group.
+
+You set up access by assigning a predefined role called `User Access principal viewer` to a user group.
+By assigning the role, users within the group are able to do the following:
+
+* View and select other users when changing owners and managing access to Kafka instances in the web console
+* Specify user names when updating the owner of a Kafka instance using the `rhoas kafka update` command of the `rhoas` CLI for {product-long-kafka}
```

Comment:
@craicoverflow -- is there command to list the users?

Reply:
No there isn't - the user list is accessible via the `rhoas kafka update` command and also the `rhoas kafka acl *` commands. cc @rkpattnaik780